Dockerfile

```diff
@@ -1,11 +1,11 @@
 # syntax=docker/dockerfile:experimental
 
 FROM opensuse/tumbleweed:latest
```

Comment:
Should we use latest or stick to a known release?  If something changes and breaks our automation, we'd most likely blame the automation.

Reply:
Known release has my vote.